### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by johnwc** [NetTopologySuite/NetTopologySuite.IO.Esri#3]
Is there any status or ETA on this project first release?

**Comment by KubaSzostak** [comment]
From my side, everything I had on my TODO list is already done.  Nevertheless, I think it would be advisable to check all open issues on [NetTopologySuite.IO.ShapeFile](https://github.com/NetTopologySuite/NetTopologySuite.IO.ShapeFile/issues) project before going forward .

**Comment by lorenh** [comment]
Will there be a NuGet package released for this soon?

**Comment by FObermaier** [comment]
@KubaSzostak I think we can do a release now.

**Comment by claasd** [comment]
Is there an ETA for (Pre-) Release?

**Comment by FObermaier** [comment]
Pre-Release versions of `NetTopologySuite.IO.Esri` are hosted on MyGet. The sources for the NetTopologySuite feed are as follows:

Version | URL
--- |---
NuGet v3 | https://www.myget.org/F/nettopologysuite/api/v3/index.json
NuGet v2 | https://www.myget.org/F/nettopologysuite/api/v2

**Comment by KubaSzostak** [comment]
Sorry for delay. I've created new PR #18. There is a [fix](https://github.com/NetTopologySuite/NetTopologySuite.IO.Esri/pull/18/commits/338411d9706830be2bef6bc5fa2b6032e69cf0ed) to possible `DbfStream` releasing issues. From my point of view once the PR is merged, it's ready to be released.

**Comment by KubaSzostak** [comment]
PR #18 merged. We are ready to release.

**Comment by FObermaier** [comment]
Here you go:
* https://github.com/NetTopologySuite/NetTopologySuite.IO.Esri/releases/tag/v1.0
* https://www.nuget.org/packages/NetTopologySuite.IO.Esri.Shapefile

**Comment by KubaSzostak** [comment]
@FObermaier, was it intentionally to add `.Shapefile` suffix to nuget package name?

**Comment by FObermaier** [comment]
No, and I'm not aware that I changed anything in that regard.
The package name has always been the same as the project name.

**Comment by KubaSzostak** [comment]
Then everything is fine. Solution name is `NetTopologySuite.IO.Esri` and project name is `NetTopologySuite.IO.Esri.Shapefile`. NuGet package name follows the project name.

**Comment by kronetix** [comment]
🎉 thanks every, we been waiting for 1.0 to implement in www.borehog.net

Regards,
Dan
